lib/BuildSystem/LaneBasedExecutionQueue.cpp

```diff
@@ -437,11 +490,11 @@ class LaneBasedExecutionQueue : public BuildExecutionQueue {
     POSIXEnvironment posixEnv;
 
     // Export a task ID to subprocesses.
-    //
-    // We currently only export the lane ID, but eventually will export a unique
-    // task ID for SR-6053.
-    posixEnv.setIfMissing("LLBUILD_TASK_ID", Twine(context.laneNumber).str());
-                          
+    auto taskID = Twine::utohexstr(context.jobID);
+    posixEnv.setIfMissing("LLBUILD_LANE_ID", Twine(context.laneNumber).str());
+    posixEnv.setIfMissing("LLBUILD_TASK_ID", taskID.str());
+    posixEnv.setIfMissing("LLBUILD_LANE_RELEASE_FD", Twine(releasePipe[1]).str());
```

Comment:
Can we pick a different name, indicative of the idea that we would potentially use this as the general purpose communication channel (SR-6053)?

Reply:
Sure.  LLBUILD_CONTROL_FD?